go/vumitools/account/models.py

```diff
@@ -29,11 +29,19 @@ class UserAppPermission(Model):
 class UserAccount(Model):
     """A user account."""
 
-    VERSION = 4
+    VERSION = 6
     MIGRATOR = UserAccountMigrator
 
+    ACCOUNT_TYPE_NORMAL = u'normal'
+    ACCOUNT_TYPE_DEVELOPER = u'developer'
```

Comment:
Really not sure if 'developer' is the right word to use here. Thoughts on a more fitting word?